# Hungarian translation for testdrive
# Copyright (c) 2010 Rosetta Contributors and Canonical Ltd 2010
# This file is distributed under the same license as the testdrive package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version: testdrive\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2010-08-06 18:04-0400\n"
"PO-Revision-Date: 2010-09-15 11:44+0000\n"
"Last-Translator: Richard Somlói <ricsipontaz@gmail.com>\n"
"Language-Team: Hungarian <hu@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2013-08-01 01:20+0000\n"
"X-Generator: Launchpad (build 16718)\n"

#: ../testdrive-gtk.desktop.in.h:1
msgid "Download and run an Ubuntu ISO in a Virtual Machine"
msgstr "Ubuntu-lemezképek letöltése, és futtatása virtuális gépben"

#: ../testdrive-gtk.desktop.in.h:2
msgid "Test Drive an Ubuntu ISO"
msgstr "Ubuntu-lemezképek tesztelése"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:1
msgid "<b>Architectures:</b>"
msgstr "<b>Architektúrák:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:2
msgid "<b>Cache:</b>"
msgstr "<b>Gyorsítótár:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:3
msgid "<b>Data Paths</b>"
msgstr "<b>Adat helye</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:4
msgid "<b>Disk Size:</b>"
msgstr "<b>Lemez mérete:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:5
msgid "<b>GB</b>"
msgstr "<b>GB</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:6
msgid "<b>Hypervisor</b>"
msgstr "<b>Hypervisor</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:7
msgid "<b>ISO Cache:</b>"
msgstr "<b>Lemezkép gyorsítótár:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:8
msgid "<b>ISO List Cache:</b>"
msgstr "<b>Lemezképlista gyorsítótár:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:9
msgid "<b>Image Cache:</b>"
msgstr "<b>Lemez gyorsítótár:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:10
msgid "<b>KVM Args:</b>"
msgstr "<b>KVM param.:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:11
msgid "<b>MB</b>"
msgstr "<b>MB</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:12
msgid "<b>Memory:</b>"
msgstr "<b>Memória:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:13
msgid "<b>Processors:</b>"
msgstr "<b>Processzorok:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:14
msgid "<b>Release:</b>"
msgstr "<b>Kiadás:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:15
msgid "<b>Repository:</b>"
msgstr "<b>Tároló:</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:16
msgid "<b>Ubuntu Flavors</b>"
msgstr "<b>Ubuntu változatok</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:17
msgid "<b>Ubuntu Releases</b>"
msgstr "<b>Ubuntu kiadások</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:18
msgid "<b>Virtual Machine Options</b>"
msgstr "<b>Virtuális gép beállításai</b>"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:19
msgid "Clean"
msgstr "Törlés"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:20
msgid "Distributions"
msgstr "Disztribúciók"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:21
msgid "Edubuntu"
msgstr "Edubuntu"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:22
msgid "General"
msgstr "Általános"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:23
msgid "KVM"
msgstr "KVM"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:24
msgid "Kubuntu"
msgstr "Kubuntu"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:25
msgid "Mythbuntu"
msgstr "Mythbuntu"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:26 ../bin/testdrive-gtk.py:47
msgid "Other"
msgstr "Egyéb"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:27
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:36
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:37
msgid "Other..."
msgstr "Egyéb…"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:28
msgid "Parallels"
msgstr "Parallels"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:29
msgid "Ubuntu"
msgstr "Ubuntu"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:30
msgid "Ubuntu Studio"
msgstr "Ubuntu Studio"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:31
msgid "Update"
msgstr "Frissítés"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:32
msgid "VirtualBox"
msgstr "VirtualBox"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:33
msgid "Virtualization"
msgstr "Virtualizáció"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:34
msgid "Xubuntu"
msgstr "Xubuntu"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:35
msgid "amd64"
msgstr "amd64"

#: ../data/ui/PreferencesTestdrivegtkDialog.ui.h:36
msgid "i386"
msgstr "i386"

#. ##################################################################
#. ####### Obtaining the settings from the Preferences Class ########
#. ##################################################################
#: ../bin/testdrive-gtk.py:136
msgid "Instancing Preferences..."
msgstr "Beállítások betöltése…"

#: ../bin/testdrive-gtk.py:150
#, python-format
msgid "<b>Release:</b> %s"
msgstr "<b>Kiadás:</b> %s"

#: ../bin/testdrive-gtk.py:151
#, python-format
msgid "<b>ISO Repository:</b> http://%s.ubuntu.com/"
msgstr "<b>Lemezkép tároló:</b> http://%s.ubuntu.com/"

#: ../bin/testdrive-gtk.py:177
msgid "Add an ISO to TestDrive"
msgstr "Lemezkép hozzáadása teszteléshez"

#: ../bin/testdrive-gtk.py:193
msgid ""
"Unable to open because 'testdrive' is not installed.\n"
"Please install testdrive: \n"
"\n"
"sudo apt-get install testdrive-cli"
msgstr ""
"A megnyitás nem sikerült. Nincs telepítve a „testdrive” csomag\n"
"Telepítse a következő paranccsal: \n"
"\n"
"sudo apt-get install testdrive-cli"

#: ../bin/testdrive-gtk.py:197
msgid "TestDrive an ISO or Disk Image"
msgstr "Lemezkép tesztelése"

#. ##################################################################
#. ## Obtains the command for the Sync process based on Protocol ####
#. ##################################################################
#: ../bin/testdrive-gtk.py:272
msgid "Obtaining the sync protocol for the specified ISO..."
msgstr "Szinkronizáló protokoll betöltése…"

#: ../bin/testdrive-gtk.py:275 ../bin/testdrive.py:325
#, python-format
msgid "Unsupported protocol [%s]"
msgstr "Nem támogatott protokoll: [%s]"

#. ##################################################################
#. #### Obtains the Virtualization Method, if not shows warnings ####
#. ##################################################################
#. Choose the virtualization engine
#: ../bin/testdrive-gtk.py:286
msgid "Obtaining the virtualization method..."
msgstr "Virtualizációs módszer betöltése…"

#: ../bin/testdrive-gtk.py:290
msgid "Your CPU supports KVM acceleration; please install KVM"
msgstr "A processzora támogatja a KVM gyorsítást, telepítse a KVM csomagot"

#: ../bin/testdrive-gtk.py:291
msgid ""
"Your CPU supports KVM acceleration; please install KVM:\n"
"\n"
"sudo apt-get install qemu-kvm"
msgstr ""
"A processzora támogatja a KVM gyorsítás, telepítse a KVM csomagot:\n"
"\n"
"sudo apt-get install qemu-kvm"

#: ../bin/testdrive-gtk.py:295
msgid ""
"Your CPU does not support acceleration; run kvm-ok for more information; "
"then install VBox"
msgstr ""
"A processzora nem támogatja a gyorsítást, futtassa a kvm-ok parancsot "
"további információért; majd telepítse a VBox csomagot"

#: ../bin/testdrive-gtk.py:296
msgid ""
"Your CPU does not support acceleration; run kvm-ok for more information;\n"
"then please install VirtualBox\n"
"\n"
"kvm-ok\n"
"sudo apt-get install virtualbox-ose"
msgstr ""
"A processzora nem támogatja a gyorsítást, futtassa a kvm-ok parancsot "
"további információért, majd telepítse a VBox-ot\n"
"\n"
"kvm-ok\n"
"sudo apt-get install virtualbox-ose"

#: ../bin/testdrive-gtk.py:303 ../bin/testdrive.py:257
msgid "Using KVM for virtual machine hosting..."
msgstr "KVM használata a virtualizációhoz…"

#: ../bin/testdrive-gtk.py:306 ../bin/testdrive.py:260
msgid "Using VirtualBox for virtual machine hosting..."
msgstr "VirtualBox használata a virtualizációhoz…"

#: ../bin/testdrive-gtk.py:309 ../bin/testdrive.py:263
msgid "Using Parallels Desktop for virtual machine hosting..."
msgstr "Parallels Desktop használata a virtualizációhoz…"

#. ##################################################################
#. ### Obtains the list of ISO available from the iso list cache ####
#. ##################################################################
#. Try to retrieve the ISO list from the cache
#: ../bin/testdrive-gtk.py:318 ../bin/testdrive.py:298
msgid "Retrieving the Ubuntu ISO list from cache..."
msgstr "Az Ubuntu-lemezképek listájának betöltése a gyorsítótárból…"

#: ../bin/testdrive-gtk.py:322 ../bin/testdrive-gtk.py:323
#: ../bin/testdrive.py:302
msgid "Unable to retrieve the Ubuntu ISO list from cache..."
msgstr ""
"Az Ubuntu-lemezképek listájának betöltése a gyorsítótárból nem sikerült…"

#. lb_cache = gtk.Label("CACHE: [%s] %s" % (time.strftime("%Y-%m-%d %H:%M:%S", time.localtime(os.path.getmtime(path))), filename))
#: ../bin/testdrive-gtk.py:441 ../bin/testdrive-gtk.py:476
#: ../bin/testdrive-gtk.py:774
#, python-format
msgid "<i>     CACHE: [%s]</i>"
msgstr "<i> GYORSÍTÓTÁR: [%s]</i>"

#: ../bin/testdrive-gtk.py:443 ../bin/testdrive-gtk.py:478
#: ../bin/testdrive-gtk.py:776
msgid "<i>     CACHE: [empty]</i>"
msgstr "<i> GYORSÍTÓTÁR: [üres]</i>"

#: ../bin/testdrive-gtk.py:506
msgid "<b><i>There are no Other ISOs yet...</i></b>"
msgstr "<b><i>Jelenleg nincsenek további lemezképek megadva…</i></b>"

#: ../bin/testdrive-gtk.py:509
msgid "<b><i>There are no ISOs for this architecture yet...</i></b>"
msgstr ""
"<b><i>Jelenleg nincsenek lemezképek ehhez az architektúrához…</i></b>"

#. Add Button
#: ../bin/testdrive-gtk.py:546
msgid "Add ISO"
msgstr "Lemezkép hozzáadása"

#. Create Buttons
#: ../bin/testdrive-gtk.py:554
msgid "Create USB Disk"
msgstr "USB indítólemez készítése"

#. bbox.add(button)
#: ../bin/testdrive-gtk.py:561
msgid "Sync"
msgstr "Szinkronizálás"

#. bbox.add(button)
#: ../bin/testdrive-gtk.py:568
msgid "Launch"
msgstr "Futtatás"

#: ../bin/testdrive-gtk.py:639
msgid "sync_iso: Thread is executing..."
msgstr "sync_iso: A szál fut…"

#: ../bin/testdrive-gtk.py:661
msgid "launch_iso: Thread is executing or syncing..."
msgstr "launch_iso: A szál fut vagy végrehajtás alatt áll…"

#: ../bin/testdrive-gtk.py:668
msgid ""
"No ISO has been selected.\n"
"\n"
"Please select an ISO to create an USB Startup Disk."
msgstr ""
"Nincs lemezkép kiválasztva.\n"
"\n"
"Válasszon egy lemezképet az USB indítólemez létrehozásához."

#: ../bin/testdrive-gtk.py:673
msgid ""
"More than 1 ISO has been selected.\n"
"\n"
"Please select only 1 ISO to continue!"
msgstr ""
"Több, mint egy lemezkép van kiválasztva.\n"
"\n"
"Csak egyet válaszon a folytatáshoz!"

#: ../bin/testdrive-gtk.py:678
msgid ""
"The specified ISO does not exist!\n"
"\n"
"Please, synchronize the ISO to continue."
msgstr ""
"A megadott lemezkép nem létezik!\n"
"\n"
"Szinkronizálja a folytatáshoz."

#: ../bin/testdrive-gtk.py:685
msgid "Unable to launch USB Creator!"
msgstr "Az USB indítólemez-készítő indítása meghiúsult!"

#. ##################################################################
#. ####### Prepare the VM to launch and return launch command #######
#. ##################################################################
#: ../bin/testdrive-gtk.py:692
msgid "Validating Virtualization Method..."
msgstr "Virtualizációs módszer érvényesítése…"

#: ../bin/testdrive-gtk.py:696 ../bin/testdrive-gtk.py:697
#: ../bin/testdrive.py:339
#, python-format
msgid "Unable to validate Virtualization Method [%s]"
msgstr "A virtualizációs módszer érvényesítése nem sikerült: [%s]"

#. self.on_info_dialog("Setting up Virtual Machine...")
#: ../bin/testdrive-gtk.py:700 ../bin/testdrive.py:341
msgid "Setting up Virtual Machine..."
msgstr "Virtuális gép beállítása…"

#: ../bin/testdrive-gtk.py:704 ../bin/testdrive-gtk.py:705
#: ../bin/testdrive.py:348
msgid "Unable to setup Virtual Machine"
msgstr "A virtuális gép beállítása nem sikerült"

#. self.on_info_dialog("Launching Virtual Machine...")
#: ../bin/testdrive-gtk.py:708
msgid "Obtaining Virtual Machine launch command..."
msgstr "Virtuális gép indító parancsának kiadása…"

#: ../bin/testdrive-gtk.py:735
#, python-format
msgid "Downloading %s..."
msgstr "Letöltés %s…"

#: ../bin/testdrive-gtk.py:797
msgid "     Running VM..."
msgstr "     Virtuális gép futtatása…"

#: ../bin/testdrive-gtk.py:853
msgid "Show debug messages"
msgstr "Hibakeresési üzenetek megjelenítése"

#: ../bin/testdrive.py:36
msgid ""
"\n"
"Welcome to Testdrive!\n"
msgstr ""
"\n"
"Üdvözöljük a Testdrive alkalmazásban!\n"

#: ../bin/testdrive.py:44
#, python-format
msgid "     +-cache--> [%s] %s"
msgstr "     +-gyorsítótár--> [%s] %s"

#: ../bin/testdrive.py:47
#, python-format
msgid "  %d. Other (prompt for ISO URL)"
msgstr "  %d. Egyéb (megadott elérési útvonalról)"

#: ../bin/testdrive.py:49
msgid ""
"\n"
"Select an image to testdrive [1]: "
msgstr ""
"\n"
"Képfájl kiválasztása teszteléshez [1]: "

#: ../bin/testdrive.py:58
msgid ""
"\n"
"ERROR: Invalid input\n"
msgstr ""
"\n"
"HIBA: Érvénytelen bemenet\n"

#: ../bin/testdrive.py:61
msgid ""
"\n"
"Enter an ISO URL to testdrive: "
msgstr ""
"\n"
"Adja meg a tesztelni kívánt képfájl elérési címét: "

#: ../bin/testdrive.py:68
msgid ""
"\n"
"ERROR: Invalid selection\n"
msgstr ""
"\n"
"HIBA: Érvénytelen kiválasztás\n"

#: ../bin/testdrive.py:72
#, python-format
msgid ""
"\n"
"ERROR: %s\n"
msgstr ""
"\n"
"HIBA: %s\n"

#: ../bin/testdrive.py:76
#, python-format
msgid "INFO: %s"
msgstr "INFÓ: %s"

#: ../bin/testdrive.py:79
#, python-format
msgid "WARNING: %s"
msgstr "VIGYÁZAT: %s"

#: ../bin/testdrive.py:90 ../testdrivegtk/AddOtherTestdrivegtkDialog.py:155
#, python-format
msgid "Invalid ISO URL [%s]"
msgstr "Érvénytelen lemezkép elérési cím: [%s]"

#: ../bin/testdrive.py:108
#, python-format
msgid ""
"Command failed\n"
"    `%s`"
msgstr ""
"Parancs hiba\n"
"    `%s`"

#: ../bin/testdrive.py:163
msgid "user configuration file (overriding default values"
msgstr "saját beállítófájl (felülírja az alapértelmezett beállításokat"

#: ../bin/testdrive.py:165
msgid "print version and system data, and exit"
msgstr "verzió és rendszer információk kiírása, majd kilépés"

#: ../bin/testdrive.py:167
msgid "get ISO image from this URL location"
msgstr "Lemezkép letöltése a megadott címről"

#: ../bin/testdrive.py:169
msgid "try to launch usb-creator for further testing"
msgstr ""
"próbálja meg elindítani az usb-creator alkalmazást  a további tesztekhez"

#: ../bin/testdrive.py:171
msgid "hardcode Ubuntu RELEASE codename"
msgstr "Az Ubuntu KIADÁS kódneve"

#: ../bin/testdrive.py:173
msgid ""
"hardcode Ubuntu flavor. Available Flavors:\n"
"\t\t     ubuntu/kubuntu/xubuntu/edubuntu/mythbuntu/ubuntustudio"
msgstr ""
"Ubuntu változatok. Elérhető változatok:\n"
"\t\t     ubuntu/kubuntu/xubuntu/edubuntu/mythbuntu/ubuntustudio"

#: ../bin/testdrive.py:177
#, python-format
msgid "version passed: %s"
msgstr "Verzió megállapítva: %s"

#: ../bin/testdrive.py:181
#, python-format
msgid "testdrive %s"
msgstr "testdrive %s"

#: ../bin/testdrive.py:204
#, python-format
msgid "config passed: %s"
msgstr "A konfiguráció megállapítva: %s"

#: ../bin/testdrive.py:214
#, python-format
msgid "Trying config in %s"
msgstr "Konfiguráció kipróbálása: %s"

#: ../bin/testdrive.py:218
#, python-format
msgid "Using configuration in %s"
msgstr "Konfiguráció használata: %s"

#: ../bin/testdrive.py:220
#, python-format
msgid "Invalid configuration [%s]"
msgstr "Érvénytelen konfiguráció [%s]"

#: ../bin/testdrive.py:250
msgid ""
"Your CPU supports KVM acceleration; please install KVM:\n"
"\t\t\tsudo apt-get install qemu-kvm"
msgstr ""
"A processzora támogatja a KVM gyorsítást, telepítse a KVM csomagot:\n"
"\t\t\tsudo apt-get install qemu-kvm"

#: ../bin/testdrive.py:253
msgid ""
"Your CPU does not support acceleration; run kvm-ok for more information; "
"then please install VirtualBox:\n"
"\t\t\tkvm-ok\n"
"\t\t\tsudo apt-get install virtualbox-ose"
msgstr ""
"A processzora nem támogatja a gyorsítást, futtassa a kvm-ok parancsot "
"további információért, majd telepítse a VBox csomagot\n"
"\t\t\tkvm-ok\n"
"\t\t\tsudo apt-get install virtualbox-ose"

#: ../bin/testdrive.py:285
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:321
#, python-format
msgid "Obtaining Ubuntu ISO list from %s..."
msgstr "Az Ubuntu-lemezképek listájának letöltése a következő helyről: %s…"

#: ../bin/testdrive.py:289
#, python-format
msgid "ERROR: Could not obtain the Ubuntu ISO list from %s..."
msgstr ""
"Az Ubuntu-lemezképek listájának letöltése sikertelen a következő helyről: %s…"

#: ../bin/testdrive.py:295
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:333
msgid "Unable to update the Ubuntu ISO list cache..."
msgstr "Az Ubuntu-lemezképek listájának gyorsítótár frissítése nem sikerült…"

#. BUG: should check disk space availability in CACHE dir
#. Update the cache
#: ../bin/testdrive.py:321
msgid "Syncing the specified ISO..."
msgstr "A megadott lemezkép szinkronizálása…"

#: ../bin/testdrive.py:328 ../testdrivegtk/AddOtherTestdrivegtkDialog.py:167
#, python-format
msgid "ISO not found at [%s]"
msgstr "A lemezkép nem található a megadott helyen: [%s]"

#. ##################
#. # Launch the VM ##
#. ##################
#: ../bin/testdrive.py:335
msgid "Validating Virtualization Method...."
msgstr "Virtualizációs módszer érvényesítése…"

#: ../bin/testdrive.py:350
msgid "Launching Virtual Machine..."
msgstr "Virtuális gép indítása…"

#: ../bin/testdrive.py:355
msgid "Unable to launch Virtual Machine"
msgstr "A virtuális gép indítása sikertelen"

#: ../bin/testdrive.py:359
#, python-format
msgid "Cleaning up disk image [%s]..."
msgstr "Lemez tisztása [%s]…"

#. Remind about cache cleanup
#: ../bin/testdrive.py:362
msgid "You may wish to clean up the cache directory..."
msgstr "A gyorsítótárként használt könyvtárak tisztítása…"

#: ../bin/testdrive.py:363
#, python-format
msgid "      %s and %s"
msgstr "      %s és %s"

#: ../bin/testdrive.py:370
msgid ""
"\n"
"Launch USB Startup Disk Creator for further testing of this ISO? [y/N] "
msgstr ""
"\n"
"Kívánja az USB indtólemez-készítőt elindítani további teszteléshez? [y/N] "

#: ../bin/testdrive.py:374
msgid ""
"\n"
"Press <enter> to exit..."
msgstr ""
"\n"
"Nyomjon <entert>-t a kilépéshez…"

#: ../data/ui/TestdrivegtkWindow.ui.h:1
msgid "ISO Repository"
msgstr "Lemezkép tároló"

#: ../data/ui/TestdrivegtkWindow.ui.h:2
msgid "TestDrive Front-end"
msgstr "TestDrive grafikusfelület"

#: ../data/ui/TestdrivegtkWindow.ui.h:3
msgid "Ubuntu Release"
msgstr "Ubuntu kiadás"

#: ../data/ui/TestdrivegtkWindow.ui.h:4
msgid "_Edit"
msgstr "S_zerkesztés"

#: ../data/ui/TestdrivegtkWindow.ui.h:5
msgid "_File"
msgstr "_Fájl"

#: ../data/ui/TestdrivegtkWindow.ui.h:6
msgid "_Help"
msgstr "_Súgó"

#: ../data/ui/TestdrivegtkWindow.ui.h:7
msgid "_View"
msgstr "_Nézet"

#: ../data/ui/AddOtherTestdrivegtkDialog.ui.h:1
msgid "<b>Description:</b>"
msgstr "<b>Leírás:</b>"

#: ../data/ui/AddOtherTestdrivegtkDialog.ui.h:2
msgid "<b>Sync Protocol:</b>"
msgstr "<b>Szinkronizáló protokoll:</b>"

#: ../data/ui/AddOtherTestdrivegtkDialog.ui.h:3
msgid "<b>URL:</b>"
msgstr "<b>URL:</b>"

#: ../data/ui/AddOtherTestdrivegtkDialog.ui.h:4
msgid "Add"
msgstr "Hozzáadás"

#: ../data/ui/AddOtherTestdrivegtkDialog.ui.h:5
msgid "Cancel"
msgstr "Mégse"

#: ../data/ui/AddOtherTestdrivegtkDialog.ui.h:6
msgid "Delete"
msgstr "Törlés"

#: ../data/ui/AddOtherTestdrivegtkDialog.ui.h:7
msgid "Edit"
msgstr "Szerkesztés"

#: ../data/ui/AddOtherTestdrivegtkDialog.ui.h:8
msgid "Other ISO List:"
msgstr "Egyéb lemezképek listája:"

#: ../testdrivegtk/AddOtherTestdrivegtkDialog.py:135
msgid "Please insert a Description"
msgstr "Nem adott meg leírást"

#: ../testdrivegtk/AddOtherTestdrivegtkDialog.py:140
msgid "Please insert a URL"
msgstr "Adjon meg elérési címet (URL)"

#: ../testdrivegtk/AddOtherTestdrivegtkDialog.py:171
msgid "No sync protocol has been selected, please select one."
msgstr ""
"Nincs szinkronizáló protokoll kiválasztva, a folytatáshoz válasszon egyet."

#: ../testdrivegtk/AddOtherTestdrivegtkDialog.py:196
msgid "Description"
msgstr "Leírás"

#: ../testdrivegtk/AddOtherTestdrivegtkDialog.py:227
msgid "Select Protocol:"
msgstr "Protokoll választás:"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:72
msgid "TestDrive Preferences"
msgstr "TestDrive beállítások"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:98
msgid "finish_initialization()"
msgstr "finish_initialization()"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:136
msgid "Select Repository:"
msgstr "Tároló kiválasztása:"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:147
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:474
msgid "Select Release:"
msgstr "Kiadás kiválasztása:"

#. Load config files for local variables
#. self.load_config_files(file)
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:211
#, python-format
msgid "Reading config file: [%s]"
msgstr "Konfiguráció beolvasása: [%s]"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:213
#, python-format
msgid "Unable to load config file [%s]"
msgstr "Konfiguráció beolvasása sikertelen: [%s]"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:268
#, python-format
msgid " of %s available."
msgstr " a(z) %s elérhetőből"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:325
#, python-format
msgid "Could not obtain the Ubuntu ISO list from %s..."
msgstr ""
"Az Ubuntu-lemezképek listájának letöltése sikertelen a következő helyről: %s…"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:329
msgid "Updating the Ubuntu ISO list cache..."
msgstr "Az Ubuntu-lemezkép lista gyorsítótárának frissítése…"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:337
msgid "get_preferences()"
msgstr "get_preferences()"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:456
#, python-format
msgid "Unable to clean up files from [%s]"
msgstr "A megadott helyről nem sikerült eltávolítani a fájlokat: [%s]"
